src/Config/Hooks.php

```diff
@@ -6,6 +6,8 @@
 
 class Hooks implements \IWPML_Action {
 
+	const PRIORITY_AFTER_DEFAULT = 20;
```

Comment:
@strategio I don't quite understand the meaning of this.

Do you mean a priority value that is after the default value of "10"?

Anyway, why you need to define it in a constant? As far as I can see, it's only used in one place (and in the test).

Reply:
Generally we don't like magic numbers. I could have chosen 11 or `PHP_INT_MAX`. What really matter in this priority is that it comes somewhere after 10 (default). IMHO, it helps to understand what we can do with this priority in the future.

Reply:
My commend it about the name, Pierre.

But sometimes I think we are a bit too fundamentalist when it comes to what are simple guidelines 👍 

The other problem here is the name of the constant that I find confusing.

Priorities are "higher" or "lower".

As you pointed out, you could have used `PHP_INT_MAX` and remove the need to define a constant which is used only in one place.

Anyway, I've already approved this PR. I just wanted to reply your last comment :)

Reply:
I think `PHP_INT_MAX` is not a proper usage here. Developers will use the default value (10) because they are just adding configuration. I don't know if there is a use case for them to use `PHP_INT_MAX` but we should keep this priority available if it's not a compulsory requirement for us.

We have this requirement sometimes, but we should have a good reason to use it IMO (see [a good example](https://github.com/Automattic/jetpack/issues/14757) where end up fighting with other devs :) ).

Reply:
Good point!